frontend/src/Season/seasonFormat.ts

```diff
@@ -0,0 +1,14 @@
+import translate from 'Utilities/String/translate';
+
+export default function seasonFormat(seasonNumber: number, separator?: string) {
+  separator ??= '- ';
+  if (seasonNumber === 0) {
+    return `${separator}${translate('Specials')}`;
+  }
```

Comment:
Should be able to set the default value like this:

```suggestion
export default function seasonFormat(seasonNumber: number, separator?: string = '- ') {

  if (seasonNumber === 0) {
    return `${separator}${translate('Specials')}`;
  }
```

Though given the super limited use, should the separator just be dropped and handled in the translation? That would be a bit more contextual and more along the lines of what we do for modal titles for providers.

Reply:
Maybe `formatSeason` or `getSeasonName`?

Reply:
I agree, getting rid of the separator is better, otherwise might screw up translations with different orders.

Reply:
only issue I face now is the translation for this modal header:
`  "OrganizeModalHeader": "Organize & Rename {season}",`
`  {translate('OrganizeModalHeader', { season: seasonNumber != null && formatSeason(seasonNumber) || '' })}`

if I add the ` - ` in the translation itself then it shows always, even when there is no seasonNumber